### PR TITLE
add script to generate chapter pdfs split into homework and reading

### PR DESCRIPTION
**Pull request by jivey**
for: https://github.com/openstax/unified/issues/2552

Already generated these and they seem to be working well for content, might be nice to merge in case we get future requests.

### CODE REVIEW

File: vite.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default {
+  server: {
+    open: 'none'
```
Comment (bethshook) [inline vite.config.js:3]: just looking at the docs, should this not be `false`?

File: scripts/generate-chapter-reading-homework-pdfs.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  const fileContents = await readFile(exercisesJsonPath, 'utf8');
+  return JSON.parse(fileContents).exercises;
+};
+
+const filterAndGroupExercises = (exercises: ExerciseData[]) => {
+  const moduleGroups = {};
+  let filteredExercises = exercises;
+
+  if (filterName === 'single-part-multiple-choice') {
+    console.log(`using filter ${filterName}`);
```
Comment (bethshook) [inline scripts/generate-chapter-reading-homework-pdfs.ts:20]: maybe this was meant to go above the conditional?

**Review by bethshook — APPROVED** [review]

File: vite.config.js (hunk 1 above)
Comment (jivey) [reply vite.config.js:3]: Hmm now I wonder if there are two ways, or maybe the vite version they use is old? This seemed to work 🤷

File: scripts/generate-chapter-reading-homework-pdfs.ts (hunk 2 above)
Comment (jivey) [reply scripts/generate-chapter-reading-homework-pdfs.ts:20]: This was more of a paranoid check so I can see it if it's running the filter... I may have run it once or twice with a typo 😆  I should probably add validation the beginning of the script if we add other filters 🤔

**Pull request merged**